### ISSUE TITLE
Bundler handling

### ISSUE THREAD

**Issue by Volzy** [Saturate/polymerase#6]
When bundling, we want to read the config for bundle-specific settings and be intelligent about how bundler processes.

[Autoprefixer](https://github.com/postcss/autoprefixer) to handle vendor prefixing. Consider promoting the CanIUse.com global usage settings rather than hardcoded legacy browsers.

**Create bundles for specific uses**
Http: Create a bundle to one file.
If config allows for Http/2: Don't bundle into one huge file, but still sanitize and minify.

Serve http/2 for http/2 supported browsers and http for non-supported.

If legacy IE is supported: Create a bundle with [css bless](http://blesscss.com/) if code exceeds IE limits.

Serve bless-bundled files for legacy IE only.

**Comment by Saturate** [comment]
I think we should be able to bundle two bundles.

So that we have for example:

- `dist/http` contains most files as one file
- `dist/http2` contains lots of single files

**Comment by Volzy** [comment]
Definitely. We still want the bundler to be clever though, so if you specify your browser requirements as "2 latest versions + Android browser 4.0", we want to create a bundle for http (should be served for Android) and http/2 for everyone else. It should not a manual task of outputting to both. Agree @Saturate?

**Comment by Saturate** [comment]
That sounds pretty cool! I agree that it should generate both...

On how we decide what to serve I don't really know, I guess it has to be a module that can switch between two paths depending on your browser, a bit like the DesignResources one we have right now.
